### PR TITLE
CLI: enter doesn't show helper anymore

### LINKED ISSUE #1449: CLI: enter shouldn't print help
It should just do nothing

### PR DESCRIPTION
**Pull request by thibault-martinez**
Closes https://github.com/iotaledger/iota-sdk/issues/1449

### CODE REVIEW

**Review by DaughterOfMars — DISMISSED** [review]

File: cli/src/account.rs [hunk 1]
```diff
@@ -94,7 +96,7 @@ pub async fn account_prompt_internal(
                 }
                 _ => {
                     // Prepend `Account: ` so the parsing will be correct
-                    let command = format!("Account: {}", command.trim());
+                    let command = format!("Account: {}", command);
```
Comment (Thoralf-M) [inline cli/src/account.rs]:
```suggestion
                    let command = format!("Account: {command}");
```
Why doesn't clippy show this?

**Review by Thoralf-M — DISMISSED** [review]

File: cli/CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - `transaction` and `transactions` indexed transactions in opposite order;
+- Enter doesn't show helper anymore;
+- Earlier trim of the input so that pre-commands (`help`, `clear`, `accounts`, ...) work even with leading/trailing spaces;
```
Comment (Alex6323) [inline cli/CHANGELOG.md:38]: I am not sure this (line 38) is a bug tbh and should be under the "Fixed" tab. To me it's more like a changed behavior? In Bash/Fish for example you can do `      clear` (so it trims), but you can't do `      cd..` (it does not trim the leading spaces) :shrug:

File: cli/CHANGELOG.md [hunk 3]
```diff
@@ -34,6 +34,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - `transaction` and `transactions` indexed transactions in opposite order;
+- Enter doesn't show helper anymore;
```
Comment (Alex6323) [inline cli/CHANGELOG.md]: I thought we don't like using temporal terms like `now`, `anymore`, etc

Also ... I have a question ... Under the "Fixed" tab, should we describe what was wrong before like I did in line 36, or should we put a description of the new fixed state there like you did here, or should we not bother and do it both ways?

File: cli/CHANGELOG.md (hunk 2 above)
Comment (Alex6323) [reply cli/CHANGELOG.md:38]: I am not against trimming btw ... I am saying that the behavior will for most people diverge from what they are used to in their terminal...and those terminals probably don't trim leading (in all cases) and trailing whitespace for a (good?) reason.

Comment (thibault-martinez) [reply cli/CHANGELOG.md:38]: You can definitely do `         cd ..` in fish, just tested it.

File: cli/src/account.rs (hunk 1 above)
Comment (thibault-martinez) [reply cli/src/account.rs]: Not sure why it doesn't 😢

File: cli/CHANGELOG.md (hunk 2 above)
Comment (thibault-martinez) [reply cli/CHANGELOG.md:38]: Idk, this really seems like a bug to me, there is absolutely no reason not to trim?

File: cli/CHANGELOG.md (hunk 3 above)
Comment (thibault-martinez) [reply cli/CHANGELOG.md]: As long as it's understandable, I'd say it doesn't matter. I'll reword.

Comment (thibault-martinez) [reply cli/CHANGELOG.md]: Done

File: cli/CHANGELOG.md (hunk 2 above)
Comment (Alex6323) [reply cli/CHANGELOG.md:38]: you're right ... when I tested I copied from here which didn't include the space in between :see_no_evil:

**Review by Alex6323 — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**